elements/urth-viz-explorer/urth-viz-explorer.html

```diff
@@ -56,6 +56,15 @@
                 notify: true,
                 observer: '_typeChanged'
             },
+
+            /**
+             * Allows user to select multiple chart components (bars in barchart, slices in pie chart, etc).
+             */
+            multiSelect: {
```

Comment:
fair enough, just that we'd need to account for a variety of other attributes as well, and might need some extensible mechanism to do so.  This widget is somewhat experimental to begin with.